wger/manager/templates/schedule/view.html

```diff
@@ -226,51 +226,53 @@ <h3 class="panel-title">
 <h4>{% trans "Adding workouts" %}</h4>
 
 <p>{% blocktrans %}Add as many workouts to the schedule as you want. You can
-edit the order by dragging and dropping them. It's also possible to add one workout more
-than once.{% endblocktrans %}</p>
+    edit the order by dragging and dropping them. It's also possible to add one workout more
```

Comment:
Even if it doesn't look nice, better don't format the texts within blocktrans, when I extract the strings for translations, they are then marked as new (because there are more spaces). Then, even if then transifex offers a suggestion to use the old translation, this is a bit more work for the translators.

(Just for the future, I'll just change these back)